.github/ISSUE_TEMPLATE/test-triage.yml

```diff
@@ -31,9 +31,9 @@ body:
     attributes:
       label: Steps to Reproduce
       value: |
-        - Commit hash where failure was observed
+        - GitHub Revision: HASH_VALUE
         - dvsim invocation command to reproduce the failure, inclusive of build and run seeds:
-          ./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i {TEST_NAME} --fixed-seed {FAILING_SEED} --build-seed {BUILD_SEED} --waves -v h
+          ./util/dvsim/dvsim.py hw/top_earlgrey/dv/chip_sim_cfg.hjson -i TEST_NAME --build-seed BUILD_SEED --fixed-seed FAILING_SEED --waves fsdb -v h
```

Comment:
should we get rid of the `-v h` by default?

Reply:
I don't mind the verbose log, but we can remove it if you think it is too spammy.

Reply:
Done!

Reply:
ah sorry, i should have clarified that was more of a question than a statement :) 
wanted to see what others think. I personally don't use `-v h` because i find it super hard to find anything.